src/webportal/src/app/job/job-view/job-detail-table.component.ejs

```diff
@@ -35,8 +35,10 @@
     <td>
       <% if (hjss === 'Waiting' || hjss === 'Running') { %>
         <button class="btn btn-default btn-xs" onclick="stopJob('<%= namespace %>', '<%= jobName %>')">Stop</button>
+        <button class="btn btn-default btn-xs" disabled>ReSubmit</button>
       <% } else { %>
         <button class="btn btn-default btn-xs" disabled>Stop</button>
+        <button class="btn btn-default btn-xs" id="resubmitjob_btn">ReSubmit</button>
```

Comment:
Use onclick handler with username as parameter (like <kbd>Stop</kbd>) is better than traversing in elements.

Reply:
Resolved